src/app/components/AccountMenu/index.tsx

```diff
@@ -60,7 +60,10 @@ function AccountMenu({ showOptions = true }: Props) {
   return (
     <div className="relative pl-2 flex justify-end w-72">
       <Menu as="div">
-        <Menu.Button className="h-full px-2 rounded-md hover:bg-gray-100 dark:hover:bg-white/10 transition-colors duration-200">
+        <Menu.Button
+          className="h-full px-2 rounded-md hover:bg-gray-100 dark:hover:bg-white/10 transition-colors duration-200"
+          aria-label={`Account menu`}
```

Comment:
you can use standard quotes`"Account menu"` unless you need to use a template string

Reply:
Okay